cluster/gce/coreos/node.yaml

```diff
@@ -93,6 +93,7 @@ coreos:
         EnvironmentFile=/etc/kube-env
         ExecStartPre=/usr/bin/rm -rf /opt/rkt
         ExecStartPre=/usr/bin/mkdir -p /opt/rkt
+        ExecStartPre=/usr/bin/mkdir -p /etc/rkt
```

Comment:
rkt uses /etc/rkt as its local config directory.  So it either needs to exist, or we can change the directory it uses by specifying --local-config=/something/else on every rkt call.